ports/stm32/lwip_inc/lwipopts.h

```diff
@@ -10,9 +10,24 @@
 
 #define LWIP_RAND() rng_get()
 
+// Increase memory for lwIP to get better performance.
+#if defined(STM32N6)
+#define MEM_SIZE                        (16 * 1024)
```

Comment:
Make larger? We have 1GB/s Ethernet onboard.

Reply:
I prefer if we use the config I made for mimxrt and alif, back then I checked it against the docs and every option is carefully calculated and it performs very well:
```
#define MEM_SIZE                        (16 * 1024)
#define TCP_MSS                         (1460)
#define TCP_OVERSIZE                    (TCP_MSS)
#define TCP_WND                         (8 * TCP_MSS)
#define TCP_SND_BUF                     (8 * TCP_MSS)
#define TCP_SND_QUEUELEN                (2 * (TCP_SND_BUF / TCP_MSS))
#define TCP_QUEUE_OOSEQ                 (1)
#define MEMP_NUM_TCP_SEG                (2 * TCP_SND_QUEUELEN)
```
Could increase the MEM_SIZE if you like.